frontend/talentsearch/src/js/components/search/deprecated/SearchForm.tsx

```diff
@@ -288,9 +283,9 @@ export const SearchForm: React.FunctionComponent<SearchFormProps> = ({
                 "Legend for the Conditions of Employment filter checklist",
             })}
             name="operationalRequirements"
-            items={operationalRequirementsSubsetV1.map((value) => ({
+            items={OperationalRequirementV1.map((value) => ({
               value,
-              label: intl.formatMessage(getOperationalRequirement(value)),
+              label: intl.formatMessage(getOperationalRequirement(value, "short")),
```

Comment:
This is where we grab the short form for the deprecated component, instead of default "full".